### PR TITLE
Team Gemini Blog post

### PR DESCRIPTION
**Pull request by akshseh**
First blog post due 10th August 2017.

### CODE REVIEW

File: blog/_posts/2017-08-10-opensource-here-we-come.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
...gular individuals, a dash of crazy, a drop of weird and a couple of undiscovered unusual ingredients makes them up.  They can be found studying in the same college of IIIT Delhi and sharing the same hostel room, adding a whole new meaning to birds of a feather flock together. Their acquaintance started with their mutual interest in coding and fascination for artificial intelligence. Slowly but steadily started bonding and came to realise that despite one being from North India and the other from South, one being a die hard Korean fan and the other a Game of Thrones fan, they are eerily similar. Their sun signs (which both are extremely fond of) also matched! While looking for the perfect team name, ‘Gemini’ which literally means ‘The Twins’ made perfect sense. In the short span of two years, they have managed to gather all sorts of memories with each other, reading for tests together, doing assignments together, bickering and laughing the next moment, watching youtube videos, heated discussions on random topics or just not seeing eye to eye with one other. They’ve been through it all and vehemently attest that their bond keeps going strong because of all their experiences together.
```
Comment (v-thakkar) [inline blog/_posts/2017-08-10-opensource-here-we-come.md]: Change "Slowly but steadily ..." -> "Slowly but steadily **they** .." , "came to realise" -> "realized"

File: blog/_posts/2017-08-10-opensource-here-we-come.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+#### [Hong phuc dang](https://github.com/hpdang)
+
+Hpdang is internationally educated business professional with focus on open technologies, developer community outreach and business development. She has extensive knowledge and technical expertise in information technologies, Open Source applications and developer community engagement.
+
+#### [Mario Behling](https://github.com/mariobehling)
+
+Mario is the co-founder of Fossasia and the founder of Susi.AI. He works as the Development lead and Fossasia summit organiser. He is also the founder of a startup named awaike. 
+
+### Supervisor
```
Comment (v-thakkar) [inline blog/_posts/2017-08-10-opensource-here-we-come.md]: Add blank line after '### Supervisor'.

File: blog/_posts/2017-08-10-opensource-here-we-come.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Mario is the co-founder of Fossasia and the founder of Susi.AI. He works as the Development lead and Fossasia summit organiser. He is also the founder of a startup named awaike. 
+
+### Supervisor 
+#### [Vaishali](https://github.com/nerdyvaishali)
+
+She is an Open Source enthusiast and works as a Linux kernel engineer at Oracle. She is also a blogger and outreachy alumni. She motivates everyone to get started with linux kernel and also takes up workshops for the same.
+
+## What we’ve done so far and the Road ahead
+
+Time indeed flies. We’ve already sifted through the large code base, understanding it, made small PRs here and there, learnt git, understand what a frontend architecture like react does and realise the effort that a community does in maintaining a project. It was one hectic month! We are really proud of the progress we’ve made in our open source journey so far.
```
Comment (v-thakkar) [inline blog/_posts/2017-08-10-opensource-here-we-come.md]: If you want to go for past tense then do that for all phrases here ["understand -> understood" ].
Also, change "realise the effort that a community does ..." -> " **Realized** **an** effort community **puts** ..."

File: blog/_posts/2017-08-10-opensource-here-we-come.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
+title: Open source, here we come!
```
Comment (v-thakkar) [inline blog/_posts/2017-08-10-opensource-here-we-come.md]: Hi, Thanks for your PR. :) Few minor changes are needed. Check out the comments below.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by akshseh** [comment]
Hi @nerdyvaishali 
Updated the blog post. Kindly review it again.

File: blog/_posts/2017-08-10-opensource-here-we-come.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+layout: post
+created_at: Monday August 10 2017
```
Comment (v-thakkar) [inline blog/_posts/2017-08-10-opensource-here-we-come.md]: Hi, can you please change the date to September 4, 2017? Change permalink field as well.

File: blog/_posts/2017-08-10-opensource-here-we-come.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+### [__Ramya:__](https://github.com/meriki)
+
+- Keeps her stuff really clean :P
+- Very lazy at times. Mastered the art of doing more in less
+- Loves fan-fiction, can’t survive without Harry Potter fanfics at least once a week
+- Gets her fat from nowhere(she hardly eats anything.. And if she does, she exercises to burn those calories)
+- Wants to know about anything and everything interesting under the sun and gets very excited every time she manages to do this. Her youtube feed is filled with random explanations about random things
+- Wants her life to be one of adventure and magic
+
+# Why Susi AI?
```
Comment (v-thakkar) [inline blog/_posts/2017-08-10-opensource-here-we-come.md]: Writing about SUSI AI project and then about sci-kit image mentors in introduction of teams looks bit odd. Can you please add the section 'our projects' or your 'What we’ve done so far and some unexpected events' section over here? There you can talk about susi AI project in 2-3 lines, your experience and what is the current situation now. And after that section you can have the same section of introducing the team as it is. You can also have 'future plan' kind of small section at the end, if you want to add that.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by akshseh** [comment]
@nerdyvaishali I made the requested changes.
Is it fine now ?

File: blog/_posts/2017-09-04-opensource-here-we-come.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+# Our Project(s)
+
+## About Susi-AI Server
+[Susi AI](http://chat.susi.ai/overview) is an open source artificial Intelligence for chatbots, personal assistants, help desks. Recognising the need to personalise chatbots Susi also provides a platform to personalise skills to a user’s requirements.
+## About Scikit-image
+[Scikit-image](http://scikit-image.org/) is an open source image processing library for the Python which includes collection of algorithms for segmentation,color space manipulation, analysis, filtering etc. It's largely written in Python, with some core algorithms written in Cython to achieve performance. Scikit-image is designed to interoperate with the Python numerical and scientific libraries NumPy and SciPy.
+
+## What we’ve done so far and some unexpected events :O 
+
+Time indeed flies. We’ve already sifted through the large code base, understood it, made small PRs here and there, learnt git, understood what a frontend architecture like react does and realized the effort a community puts in maintaining a project.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-opensource-here-we-come.md]: Post looks better to me now. Only one concern here. It's not clear about which project you are talking here. It's totally fine to mention which project was assigned to you previously, what you learnt/worked on and which project you will be working on now. Can you please add project names and things accordingly? Thanks.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**